### ISSUE TITLE
Hover colour on pagination `<Pagination />` not visible on certain monitors

### ISSUE THREAD

**Issue by vnys** [equinor/design-system#1685]
https://equinor.slack.com/archives/CJT20H1B9/p1636539531059400

_“We want to change this because of a request from the customer in Equinor after our last demo. On the monitors in the Equinor offices, both wall mounted and at workstations, it is basically impossible for them to see where they have navigated, so they requested that we changed it to a higher contrast color. The same goes for "Mist blue" and "Spruce wood", they're very hard to see at the customer location.
The thought was to change it to "Slate blue" to keep it within the general color scheme. Since "Moss green" is used for buttons we don't want those two schemes to compete, we want it to be very clear what's navigation and what's an action. This is also a solution with ~10 users, so we prioritize usability over any other requirement.
Hope that explains a bit? :slightly_smiling_face:
You're of course also welcome up to TSB3 to see how it looks in real life on the monitors that are deployed there, they aren't exactly good when it comes to color reproduction.”_

CC: @dthgs, @kjetvin

**Comment by vnys** [comment]
This is a returning problem with spruce wood, lichen green and mist blue. Nice colours, but pretty useless as state indicators if you can’t see them. Should we increase the contrast in the current theme? Or create a [high contrast theme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast)?

_I think we need two things here:_
1) A short-term solution for this specific project
1) A long-term solution for everyone else who has similar issues

CC: @benteljones, @lucasveil, @BirteThornquist

**Comment by mimarz** [comment]
This use-case looks to be relevant to #1426 also?

**Comment by vnys** [comment]
> This use-case looks to be relevant to #1426 also?

Definitely. It would be interesting to measure where the limit is, if possible. Creating a test-page with various tints to test out on the clients monitors for example.

**Comment by kjetvin** [comment]
I can point you in the direction of some really bad monitors to test on in Forus Øst 😊

**Comment by vnys** [comment]
Temporary solution → https://codesandbox.io/s/eds-pagination-01-jjy25?file=/src/App.js
